docs/phylum_extension_add.md

```diff
@@ -0,0 +1,17 @@
+---
+title: phylum extension add
+category: 6255e67693d5200013b1fa3e
+hidden: true
+---
+To install an extension, run the following command:
+
+```sh
+phylum extension add path/to/extension
+```
+
+The extension will be installed under `$XDG_DATA_HOME/phylum/extensions/<ext_name>`.
```

Comment:
This should also mention the default location of `$HOME/.local/share/phylum/extensions/<ext_name>` if `$XDG_DATA_HOME` isn't set.